test/helper_test.exs

```diff
@@ -41,6 +41,34 @@ defmodule HelperTest do
     assert {:ok, %{message_count: 0}} = Queue.delete(meta[:chan], queue_name_errored)
   end
 
+  test "configuration without an exchange and with routing key should not raise an error", meta do
+    queue_name = rand_name()
+    routing_key_name = rand_name()
+    exchange_name = rand_name()
+
+    queue_name_errored = "#{queue_name}_errored"
+
+    assert :ok =
+             Helper.declare(meta[:chan], %{
+               exchanges: [
+                 %{name: exchange_name, opts: [durable: true], routing_keys: [routing_key_name], type: :topic}
+               ],
+               opts: [
+                 durable: true,
+                 arguments: [
+                   {"x-dead-letter-exchange", :longstr, ""},
+                   {"x-dead-letter-routing-key", :longstr, routing_key_name}
```

Comment:
mm non ho capito il motivo di "allargare le maglie" del controllo.. 

qua dovrebbe essere questo il setup che va in OK
```suggestion
                   {"x-dead-letter-routing-key", :longstr, queue_name_errored}
```
ed essere l'unico caso accettato di "DLX vuoto, mi appoggio al default exchange" (revertando quindi le modifiche fatte contestualmente all'helper).. 

altrimenti l'effetto sarebbe avere su rabbit
- una coda "per i messaggi di errore" chiamata `{routing_key_name}` (anziché `{queue_name}_errored`
- nessun binding impostato per quella coda
- i messaggi in errore vengono ri-emessi con chiave `routing_key_name`
- nessun exchange li raccoglie, vanno nel default exchange
- che li mette nella coda "per i messaggi di errore"

che in se e per sé va bene, ma porta un rischio di conflitti con i nomi delle code esistenti e riduce la _clarity_ rispetto a forzare tutte le code di errore al nome `_errored` 🤷🏻‍♂️